js/app.js

```diff
@@ -78,13 +77,34 @@ function render() {
   console.log(totalClicks);
   comparisonValues.splice(0,3);
   if (totalClicks < 1) {
-    ulEl.removeEventListener('click', render);
-    //call stats method
+    ulEl.removeEventListener('click', handleClicks);
+    listCreator();
   }
 }
 
+//Sam method for handling clicks for the pictures.
+function handleClicks(event) {
+  for (var i = 0; i < ImageCreator.allImages.length; i++) {
+    if (event.target.alt === ImageCreator.allImages[i].title) {
+      ImageCreator.allImages[i].clicks += 1;
```

Comment:
Note that you can use `ImageCreator.allImages[i].clicks++` to do the same thing.